crates/core_arch/src/x86_64/bt.rs

```diff
@@ -0,0 +1,102 @@
+#[cfg(test)]
+use stdsimd_test::assert_instr;
+
+/// Returns the bit in position `b` of the memory addressed by `p`.
+#[inline]
+#[cfg_attr(test, assert_instr(bt))]
+#[unstable(feature = "simd_x86_bittest", issue = "0")]
+pub unsafe fn _bittest64(p: *const u64, b: u64) -> u8 {
+    let r: u8;
+    asm!("btq $2, $1\n\tsetc ${0:b}"
+         : "=r"(r)
+         : "*m"(p), "r"(b)
+         : "cc", "memory");
+    r
+}
+
+/// Returns the bit in position `b` of the memory addressed by `p`, then sets the bit to `1`.
+#[inline]
+#[cfg_attr(test, assert_instr(bts))]
+#[unstable(feature = "simd_x86_bittest", issue = "0")]
+pub unsafe fn _bittestandset64(p: *mut u64, b: u64) -> u8 {
+    let r: u8;
+    asm!("btsq $2, $1\n\tsetc ${0:b}"
+         : "=r"(r), "+*m"(p)
+         : "r"(b)
+         : "cc", "memory");
+    r
+}
+
+/// Returns the bit in position `b` of the memory addressed by `p`, then resets that bit to `0`.
+#[inline]
+#[cfg_attr(test, assert_instr(btr))]
+#[unstable(feature = "simd_x86_bittest", issue = "0")]
+pub unsafe fn _bittestandreset64(p: *mut u64, b: u64) -> u8 {
+    let r: u8;
+    asm!("btrq $2, $1\n\tsetc ${0:b}"
+         : "=r"(r), "+*m"(p)
+         : "r"(b)
+         : "cc", "memory");
+    r
+}
+
+/// Returns the bit in position `b` of the memory addressed by `p`, then compliments that bit.
```

Comment:
*complements.

Then again, I personally think "inverts" would be clearer in this context.

Reply:
Fixed. I copied (without reading apparently) from the Intel link you added.